README.md

```diff
@@ -0,0 +1 @@
+# Go Packages
```

Comment:
we should get rid of this README file, it'll only conflict with the real one

Reply:
removed